incubator/virtualcluster/pkg/vn-agent/server/route.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package server
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+
+	"github.com/emicklei/go-restful"
+	"k8s.io/apimachinery/pkg/util/proxy"
+	"k8s.io/klog"
+)
+
+// InstallHandlers set router and handlers.
+func (s *Server) InstallHandlers() {
+	ws := new(restful.WebService)
+	ws.
```

Comment:
can this paas gofmt? just make this one line?

Reply:
alright. But it really could pass the gofmt.